### PR TITLE
Image (service) documentation

### PR DESCRIPTION
**Pull request by gabrielhase**

### CODE REVIEW

**Comment by gabrielhase** [comment]
@mthpvg fyi

**Comment by gabrielhase** [comment]
@rebelact sorry, forgot, also fyi 😄

**Comment by rebelact** [comment]
Awesome! 💯

File: public/guides/images/image-services.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+# Image Services
+
+## Overview
+
+Livingdocs allows users to upload pictures and then subsequently shows them. There are 2 parts here:
+1. the storage and delivery of the images
```
Comment (marcbachmann) [inline public/guides/images/image-services.md]: Do you want to split those up?
Delivery and storage are most times completely different. Delivery needs to be scalable, storage (image upload) doesn't.

Comment (marcbachmann) [inline public/guides/images/image-services.md]: Can you link to the image proxy (which also does storage and is basically the additional layer I meant) documentation.

Comment (gabrielhase) [inline public/guides/images/image-services.md]: I don't really know about an upload proxy. Can you link to it? 😄

**Pull request merged**